cli.sh

```diff
@@ -86,7 +86,7 @@ case $1 in
     ;;
   ap)
     checkroot
-    ap "$2" "$3" "$4"
+    ap "$2" "$3" "$4" "$5"
```

Comment:
hmm no `--ip` check

which mean in the case of an open hotspot (no password)
`treehouses ap internet treehouse --ip 192.168.3.0`
we have a problem